test/runtests.jl

```diff
@@ -6,6 +6,7 @@ include("worldclim-weather.jl")
 include("earthenv-landcover.jl")
 include("earthenv-heterogeneity.jl")
 include("chelsa-bioclim.jl")
```

Comment:
tests for `WorldClim`?

Reply:
no, because the zip files don't seem to contain any data... this is weird and I need to dig into it.

Reply:
Ok no worries, that sounds annoying. What kind of zip is it?

You could ould also move WorldClim Future to another PR if you want to get CHELSA merged

Reply:
I might do that. Let me think about the types a bit more,  and I'll get back with some infos tomorrow.